app/app.go

```diff
@@ -20,11 +20,11 @@ import (
 type App struct {
 	cfg                  *config.Config
 	dataSourceRepository *physical.DataSourceRepository
-	out                  output.Output
+	out                  interface{}
```

Comment:
Explain this please, `interface{}` is the least self-explanatory thing there is, so it deserves a comment

Reply:
Fixed, I overlooked this.